components/icon/index.en-US.md

```diff
@@ -0,0 +1,125 @@
+---
+category: Components
+chinese: 图标
+type: Basic
+english: Icon
+toc: false
+---
+
+Meaningful vector graphics, each one can defeat an enemy.
+
+## The specification of naming icons
+
+We provide semantic name for every icon, and naming rules are as follows: 
+
+- Scanning line icon has the similar name with its solid one，but it's distinguished by `-o`, for example, `question-circle`(solid) and `question-circle-o`(scanning line);
+
+- Naming sequence：`[icon's name]-[shape, optional]-[Scanning line or not]-[direction, optional]`.
+
+## How to use
+
+Use tag `<Icon />` to define an icon component, and appoint its type, for example:
+
+```html
+<Icon type="link" />
+```
+
+Finally, it will be rendered as follow:
+
+```html
+<i class="anticon anticon-${type}"></i>
+```
+
+## Local deployment
+
+When icon component uses [iconfont.cn](http://iconfont.cn), by default, the public network can access it. Suppose you need a local deployment, you can refer to [example](https://github.com/ant-design/antd-init/tree/master/examples/local-iconfont)。
```

Comment:
By default, icon components uses [iconfont.cn](http://iconfont.cn), publicly available repository of a huge set of icons. In case you need to use a locally deployed version of the icon font, you can refer to [this example](https://github.com/ant-design/antd-init/tree/master/examples/local-iconfont).